config.env.template

```diff
@@ -1,3 +1,27 @@
 # GETH compliant RPC APIs for layer 1 & 2 blockchains
 L1_RPC_ENDPOINT=""
-L2_RPC_ENDPOINT=""
\ No newline at end of file
+L2_RPC_ENDPOINT=""
+
+# Environemnt
+ENV=local                               # local,development,production
+
+# Custom Logger Configs 
+LOGGER_USE_CUSTOM=0                     # 0 or 1
+LOGGER_LEVEL=-1                         # -1 (debug), 0 (info), 1 (warn), 2 (error), 3 (dpanic), 4 (panic), 5 (fatal)
+LOGGER_DISABLE_CALLER=0                 # 0 or 1
+LOGGER_DISABLE_STACKTRACE=0             # 0 or 1
+LOGGER_ENCODING=console                 # json,console
+LOGGER_OUTPUT_PATHS=stderr              # comma separated paths
+LOGGER_ERROR_OUTPUT_PATHS=stderr        # comma separated paths
```

Comment:
Awesome job on enumerating all the potential value options

Reply:
thx <3